lighthouse-core/gather/driver/wait-for-condition.js

```diff
@@ -258,6 +262,69 @@ function waitForCPUIdle(session, waitForCPUQuiet) {
   };
 }
 
+/* c8 ignore start */
+
+/**
+ * This function is executed in the page itself when the document is first loaded.
+ *
+ * Used by _waitForCPUIdle and executed in the context of the page, updates the ____lastLongTask
+ * property on window to the end time of the last long task.
+ */
+function registerPerformanceObserverInPage() {
+  // Do not re-register if we've already run this script.
+  if (window.____lastLongTask === undefined) return;
```

Comment:
```suggestion
  if (window.____lastLongTask !== undefined) return;
```

doh, it was the suggestion I didn't have locally ;)